Demo/build.gradle

```diff
@@ -16,12 +17,11 @@ android {
         applicationId "com.paypal.android"
         minSdkVersion 21
         targetSdkVersion 33
-        versionCode 1
+        versionCode 2
```

Comment:
Should there be a step in this workflow to bump this? It will fail if it's not a higher number than previous version right?

Reply:
Ended up updating the script to do this automatically 👍.